roles/bootstrap/tasks/install_python.yaml

```diff
@@ -0,0 +1,33 @@
+- name: Install python 3.8.16
+  block:
+    - name: Test if Python 3.8 is already installed
+      shell:
+        cmd: which python3.8
+      register: python_installed
+      failed_when: "python_installed.rc not in [0,1]"
+    - name: "Manually install Python 3.8"
+      block:
+        - name: "add packages required to install python"
+          apt:
+            state: present
+            name:
+              - build-essential
+              - checkinstall
+              - libncursesw5-dev
+              - libssl-dev
+              - libsqlite3-dev
+              - tk-dev
+              - libgdbm-dev
+              - libc6-dev
+              - libbz2-dev
+              - libffi-dev
+              - zlib1g-dev
+        - name: "download and build"
+          become: true
+          become_method: sudo
+          block:
+            - shell: "cd /opt && wget https://www.python.org/ftp/python/3.8.16/Python-3.8.16.tgz"
```

Comment:
J'ai l'impression qu'on a des built-in ansible pour faire ça, du genre : 
`ansible.builtin.get_url` et `ansible.builtin.unarchive`

Par ailleurs on peut ajouter une étape pour supprimer l'archive ?